modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/SignClient.scala

```diff
@@ -28,9 +27,9 @@ object SignClient {
       def getRegistrationRequest: PeerResponse[F, RegistrationRequest] =
         PeerResponse[F, RegistrationRequest]("registration/request")(client)
 
-      def joinRequest(jr: JoinRequest): PeerResponse[F, Unit] =
-        PeerResponse[F, Unit]("cluster/join", POST)(client) { (req, c) =>
-          c.successful(req.withEntity(jr)).void
+      def joinRequest(jr: JoinRequest): PeerResponse[F, Boolean] =
```

Comment:
Maybe instead of a `Boolean` continue using `Unit` and change the response code from `200` to `409`. It may be useful if someone would create additional tooling around the CLI API (`curl` will return error exit code on responses outside of the 2xx family).

Reply:
Boolean isn't received from the endpoint. Take a look how `client.successful` works. I just expect that the request succeeded and it returns me yes or no.

Reply:
Ah, alright then, I misread that!